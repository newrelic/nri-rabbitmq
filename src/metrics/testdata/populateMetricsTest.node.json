[
    {
        "partitions": [
            "one",
            "two"
        ],
        "mem_alarm": true,
        "name": "node1",
        "running": true,
        "sockets_used": 523
    }
]