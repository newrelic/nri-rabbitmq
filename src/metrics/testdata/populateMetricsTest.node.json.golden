{"displayName":"node1","entityName":"node:node1","event_type":"RabbitmqNodeSample","node.fileDescriptorsUsedSockets":523,"node.hostMemoryAlarm":1,"node.partitionsSeen":2,"node.running":1,"rabbitmqClusterName":"testClusterName","reportingEndpoint":"foo:8000"}